zkabacus-crypto/src/customer.rs

```diff
@@ -346,6 +346,7 @@ pub struct Started {
 /// This is sent as part of zkAbacus.Pay.
 #[derive(Debug)]
 #[non_exhaustive]
+#[allow(missing_copy_implementations)]
```

Comment:
I think this is a question for @marsella.

Reply:
I thought I was getting a warning when it wasn't there. Are you saying the default behavior is to not warn if `Copy`s are missing? 

It's missing because we don't want it to be easy to copy this and store it as an outdated channel status (which you could then use to try to close the channel).

Reply:
In any case, I think I'd like to add doc comments where we have a security reason for not implementing Copy. Otherwise I am not sure this decision would be lasting?

Reply:
> Are you saying the default behavior is to not warn if Copys are missing?

Yes, we override it https://github.com/boltlabs-inc/libzkchannels-crypto/blob/main/zkchannels-crypto/src/lib.rs#L9

Reply:
I suggest we make this an issue on its own and not part of this PR.

Reply:
That sounds like a good plan!